BlueShift-iOS-SDK/BlueShiftDeviceData.m

```diff
@@ -30,7 +30,10 @@ - (NSString *)deviceUUID {
             [defaults setObject:UUID forKey: @"BlueshiftDeviceIdSourceUUID"];
             deviceUUID = [UUID copy];
         }
-    } else {
+    } else if (_blueshiftDeviceIdSource && _blueshiftDeviceIdSource == BlueshiftDeviceIdSourceIDFVBundleID && [[NSBundle mainBundle] bundleIdentifier] != nil) {
```

Comment:
one suggestion to avoid duplicate nil check.
do the nil check outside the if-else ladder and then compare the enum value.

Reply:
Resolved by replacing if else ladder by switch.